includes/class-ssp-frontend.php

```diff
@@ -284,7 +284,7 @@ public function episode_meta( $episode_id = 0, $context = 'content' ) {
 
 			// Show audio player if requested
 			if( $show_player ) {
-				$meta .= '<div class="podcast_player">' . $this->media_player( $file, $episode_id ) . '</div>';
+				$meta .= '<div class="podcast_player" style="margin: 15px 0;">' . $this->media_player( $file, $episode_id ) . '</div>';
```

Comment:
I'm not the biggest fan of inline styling, but I understand why this has been placed here. Would it be possible to add something like a 'player.css' file to be enquired on the front end of the site, for all custom CSS to be enqueued there. This way if any future changes need to be made we only need search from them in one place.

Reply:
Not a problem. I've created a player.css file to hold this instead :)